src/lib/viewers/controls/color-picker/__tests__/ColorPickerControl-test.tsx

```diff
@@ -36,7 +37,7 @@ describe('ColorPickerControl', () => {
             toggleButton.simulate('click');
             expect(getToggleButton(wrapper).hasClass('bp-is-active')).toBe(true);
 
-            toggleButton.simulate('blur');
+            toggleButton.simulate('blur', {});
```

Comment:
Is the empty object needed?

Reply:
Yeah, otherwise the test will throw error for getting `relatedTarget` from `undefined`